### ISSUE TITLE
#Aaron for bg picture

### ISSUE THREAD

**Issue by bajracharya1** [WebDevBootcamp/AJAP#14]
Aaron, if your still looking for bg picture, u can use the one that Audrey have on her folder
/treehouseCodeschool/computerScene.jpeg 

its under her branch.

And close this issue..

**Comment by Klammertime** [comment]
Or this jackpot:
https://unsplash.com/

It's where I got it and get all of my images lately.

**Comment by nw** [comment]
Nice resource

**Comment by bajracharya1** [comment]
cool... that sure is a jackpot

**Comment by amgoodman5** [comment]
Yes, I'll have a look at that site. Seems  like there is a bunch of great material!

**Comment by Klammertime** [comment]
And if you can't decide and want several of the same type to load you can look here:
http://unsplash.it/

I know this might be crazy but its a lot of fun.
